lib/services/local/presence.go

```diff
@@ -138,6 +139,85 @@ func (s *PresenceService) DeleteNamespace(namespace string) error {
 	return trace.Wrap(err)
 }
 
+// GetServerInfos returns a stream of ServerInfos.
+func (s *PresenceService) GetServerInfos(ctx context.Context) stream.Stream[types.ServerInfo] {
+	const pageSize = 100
```

Comment:
I think we can do 1000 without too much strain.

```suggestion
	const pageSize = apidefaults.DefaultChunkSize
```

Reply:
Agreed - and keeps this consistent with other methods.